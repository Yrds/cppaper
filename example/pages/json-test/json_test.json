{
  "html": "html value",
  "test": "test value"
}
