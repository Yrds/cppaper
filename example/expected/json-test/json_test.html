<!DOCTYPE html>
<html>
  <head>
    <meta charset="UTF-8" />
    <meta name="viewport" content="width=device-width" />
    <title>Yuri Santos - json_test</title>
  </head>
  <body>
    test value
  </body>
</html>
